uber/templates/base-admin.html

```diff
@@ -11,5 +11,12 @@
 {%  endblock %}
 
 {% block additional_styles %}
+   <!-- inside additional_styles -->
    <link rel="stylesheet" type="text/css" href="../static_views/additional-styles.css" />
+   <link rel="stylesheet" href="../static/styles/bootstrap.min.css" />
```

Comment:
Our JavaScript is packaged into a file called `combined.css`, which is already included in the template this template inherits from. I definitely don't expect you to know how to add new plugins to the package (since there's no documentation on it and it's a complex process), but the important thing to know is that Bootstrap and jQuery are included -- you shouldn't need to have this line or the boostrap or jquery folders.